localization/english/bookmark/bookmark_603_l_english.yml

```diff
@@ -1,11 +1,27 @@
+
+ #Anasterian Sunstrider
+ anasterian_character:0 "High King Anasterian"
+ anasterian_character_desc:0 "For centuries already have Anasterian Sunstrider ruled over Quel'Thalas as its king, heir to the line of Dath'remar. Wise but proud, the old king values independance and tranquility and as such withdrawed entirely from the Alliance after victory was acquired against the Horde during the second war. With the orcs and trolls repelled from his frontiers, Anasterian is confident that his kingdom's prosperity and magical superiority will assure his people's safety and happiness in the haven that are the thalassian forests ; numerous elves, including crown-prince Kael'Thas, however, remain sympathethic to the Alliance and believe it is the elves's duty to help their former allies in these difficult, changing times. Even the secluded elves can't ignore the events of the world forever. What destiny awaits the elven king and his people ? To prosper secluded in their magical kingdom ? To unite with their formers allies against a darker threat ? To fall into oblivion alone...Or perhaps to take an even greater place in the world ? "
+
+ #Ner'zhul
+ nerzhul_character:0 "Ner'zhul, the Lich King"
+ nerzhul_character_desc:0 "Once, Ner'Zhul was an orc. As chieftain of the Shadowmoon clan and elder shaman, he was the spiritual leader of the orcs, deceived to unite them into the warmongering Horde by the demon-lord Kil'Jaeden as a tool of war for the Burning Legion. When he tried to oppose his demonic master, Ner'Zhul was betrayed and replaced by his apprentice, Gul'dan, and when the Horde was finally defeated and Draenor destroyed, Ner'Zhul tried to flee, only to end in the claws of the Legion's minions. For his treason, he was remade into a spectral being, the Lich King, a new weapon of war against Azeroth. Sent to the continent of Northrend to prepare for his mission, the Lich King discovered his immense powers and unleashed them under the vigilant supervisions of Kil'Jaeden's agents. Ner'Zhul has built the base of a new order, to the Legion, he will be the vanguard of their invasion, and to the people of Azeroth, a god of death about to make a new world reborn in undeath. Will Ner'Zhul succeed in his task to ensure the Legion's return ? Will he be stopped before such tragedy could occur...Or will he achieve a different destiny, as his own master, and ruler of both the deads and the world ?"
```

Comment:
Too big, and cuts off at the last sentence
![image](https://user-images.githubusercontent.com/74790101/223279923-d6790e88-1e75-4807-9a57-40f77fc1b0c5.png)